src/Smart.Design.Console/Models/CommandResponse.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Smart.Design.Console.Models
+{
+    public class CommandResponse
+    {
+        public bool IsSuccess => !Errors.Any();
+        private string SuccessMessage;
+
+        public string GetResponseMessage()
+        {
+            if (IsSuccess) return SuccessMessage;
+
+            StringBuilder stringBuilder = new("Exception Stacks");
+            Errors.ForEach(error => stringBuilder.AppendLine($"exception with message => {error}"));
+            return stringBuilder.ToString();
+        }
+
+        private List<Exception> Errors = new();
```

Comment:
Is it normal to declare an attribute here instead of at the beginning of the class ?

Reply:
You mean the declaration of the private field ?

Reply:
Yisss